diff --git a/esm/vs/base/common/platform.js b/esm/vs/base/common/platform.js
index d507327daec3cdf5027bd886f898c364c5433ecd..25fe1dd9d08b75694e9e90c62391576f487410b0 100644
--- a/esm/vs/base/common/platform.js
+++ b/esm/vs/base/common/platform.js
@@ -26,7 +26,7 @@ if (typeof $globalThis.vscode !== 'undefined' && typeof $globalThis.vscode.proce
     // Native environment (sandboxed)
     nodeProcess = $globalThis.vscode.process;
 }
-else if (typeof process !== 'undefined') {
+else if (typeof process !== 'undefined' && typeof process?.versions?.node === 'string') {
     // Native environment (non-sandboxed)
     nodeProcess = process;
 }
